docs/development/getting_started.rst

```diff
@@ -46,32 +46,50 @@ which is signficantly out of date and known not to work with SecureDrop (`Github
 .. _`Vagrant Downloads page`: https://www.vagrantup.com/downloads.html
 .. _`GitHub #932`: https://github.com/freedomofpress/securedrop/pull/932
 
+.. warning:: We do not recommend installing vagrant-cachier. It destroys apt’s
+            state unless the VMs are always shut down/rebooted with Vagrant,
+            which conflicts with the tasks in the Ansible playbooks. The
+            instructions in Vagrantfile that would enable vagrant-cachier are
+            currently commented out.
+
 Either way, once you've installed Vagrant you should run:
 
 .. code:: sh
 
     sudo dpkg-reconfigure virtualbox-dkms
 
 Finally, install Ansible so it can be used with Vagrant to automatically
-provision VMs. We recommend installing Ansible from PyPi with ``pip`` to ensure
-you have the latest stable version.
+provision VMs. We recommend installing Ansible from PyPi with ``pip``, which
+allows you to specifically install the version of Ansible available in Tails,
+currently 1.7.2 (5/6/16):
 
 .. code:: sh
 
     sudo apt-get install python-pip
-    sudo pip install -U ansible
 
-If you're using Ubuntu, you can install a sufficiently recent version of
-Ansible from backports (if you prefer): ``sudo apt-get install
-ansible/trusty-backports``
+Since the version of ansible required to provision SecureDrop VMs may flux out
+of sync with the version in your distro's repos, we recommend installing the
+latest stable ansible to a virtual environment using `virtualenvwrapper` or
```

Comment:
> Anyway, I'm going to reinstate 1.8.4 for now.

This frustrates me deeply, but I agree with the determination: tell devs to use 1.8.4, because that's what we're using. We always test on hardware, in Tails, prior to a release, so we'll still catch any version-specific tangles during that process. It's good to keep in mind that we're intentionally conservative with module choice in order to retain support for older versions of Ansible—see #1146 for tracking more modern versions of Ansible for admins.